couchbase documentation (#2253)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/couchbase.md

---
#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
title: Couchbase credentials
description: Documentation for Couchbase credentials. Use these credentials to authenticate Couchbase in n8n, a workflow automation platform.
contentType: integration
priority: medium
---

# Couchbase credentials

You can use these credentials to authenticate the following nodes:

- [Couchbase](/integrations/builtin/app-nodes/n8n-nodes-base.couchbase/)

## Prerequisites

> - Create a user account with the appropriate permissions on a [Couchbase](https://www.couchbase.com/){:target=_blank .external-link} server.

Reviewer: ```suggestion
- [Database access credentials](https://docs.couchbase.com/cloud/clusters/manage-database-users.html#create-database-credentials){:target=_blank .external-link} (username and password) with the appropriate permissions on a [Couchbase Capella](https://cloud.couchbase.com/){:target=_blank .external-link} cluster or a [Couchbase Server](https://docs.couchbase.com/server/current/install/install-intro.html) cluster.
```

> If you are setting up Couchbase from scratch, create a cluster and a bucket. Refer to the [Couchbase documentation](https://docs.couchbase.com/home/index.html){:target=_blank .external-link} for more detailed instructions on these steps.

Reviewer: ```suggestion
If you are setting up Couchbase from scratch, you'll have to create a cluster and a bucket. Refer to the [Couchbase documentation](https://docs.couchbase.com/home/index.html){:target=_blank .external-link} for more detailed instructions on these steps.
```

## Supported authentication methods

> - Database connection - Values

Reviewer: I think we're good to just say "Database connection", since we only support one method of DB connection (ref for my thinking [here](https://docs.n8n.io/integrations/builtin/credentials/mysql/))
 ```suggestion
- Database connection
```

## Related resources

Refer to the [Couchbase documentation](https://docs.couchbase.com/home/index.html){:target=_blank .external-link} for more information about the service.

> ## Using database connection - Values

Reviewer: ```suggestion
## Using Database Connection
```

To configure this credential, you'll need:

- The **Connection String** for the database

> - The **Username**

Reviewer: ```suggestion
- A **Username**, with the appropriate permissions.
```

> - A user **Password**

Reviewer: ```suggestion
- A **Password** for that user
```

> - - A **Bucket**, **Scope**, and **Collection**

Reviewer: Do we mean to have the double `-` `-` here?
